src/lib/migration-chunks/validation/editor-layout.ts

```diff
@@ -192,23 +238,23 @@ class DuplicateDeletes implements EditorLayoutValidation {
 
 class InvalidFielGroupIdChange implements EditorLayoutValidation {
   validate (intent: Intent, { remoteFieldGroups, createdFieldGroups }: ValidationContext): string | string[] {
-    if (intent.getRawType() !== 'contentType/changeEditorLayoutFieldGroupId') {
+    if (!intent.isFieldGroupIdChange()) {
       return
     }
 
     const changeErrors = editorLayoutErrors.changeFieldGroupId
-    const changeIntent = intent as EditorLayoutChangeFieldGroupId
-    const contentTypeId = changeIntent.getContentTypeId()
-    const fieldGroupId = changeIntent.getFieldGroupId()
-    const newFieldGroupId = changeIntent.getNewFieldGroupId()
+
+    const contentTypeId = intent.getContentTypeId()
+    const fieldGroupId = intent.getFieldGroupId()
+    const newFieldGroupId = intent.getNewFieldGroupId()
 
     if (!fieldGroupId) {
       return changeErrors.MISSING_FIELD_GROUP_ID()
     }
-    if (!newFieldGroupId) {
+    if (newFieldGroupId === undefined) {
```

Comment:
Made the check more specific so empty string will trigger the _InvalidFieldGroupId_ validation.